paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/action/cdc/Expression.java

```diff
@@ -608,4 +618,37 @@ public String eval(String input) {
             return value;
         }
     }
+
+    /** Get current timestamp. */
+    final class AuditTimeExpression implements Expression {
+
+        private final boolean insertOnly;
+        private final String fieldReference;
+
+        public AuditTimeExpression(boolean insertOnly) {
+            this.insertOnly = insertOnly;
+            this.fieldReference = SpecialFields.VALUE_KIND.name();
+        }
+
+        @Override
+        public String fieldReference() {
+            return this.fieldReference;
+        }
+
+        @Override
+        public DataType outputType() {
+            return DataTypes.TIMESTAMP();
+        }
+
+        @Override
+        public String eval(String input) {
+            RowKind rowKind = RowKind.fromShortString(input);
+            String now = java.sql.Timestamp.valueOf(LocalDateTime.now()).toString();
+            if (rowKind == RowKind.INSERT) {
+                return now;
+            }
+
+            return insertOnly ? null : now;
```

Comment:
Copy from https://github.com/apache/paimon/pull/5086 :
So if a new input update the record, the create_time will become null?